src/main/java/net/pms/encoders/FFMpegVideo.java

```diff
@@ -356,13 +356,17 @@ public FFMpegVideo(PmsConfiguration configuration) {
 					transcodeOptions.add("-an");
 				} else if (type() == Format.AUDIO) {
 					// Skip
-				} else if (renderer.isTranscodeToAAC()) {
-					transcodeOptions.add("-c:a");
-					transcodeOptions.add("aac");
 				} else {
-					if (!customFFmpegOptions.contains("-c:a ")) {
-						transcodeOptions.add("-c:a");
-						transcodeOptions.add("ac3");
+					if (!customFFmpegOptions.matches(".* -(c:a|codec:a|acodec) .*")) {
```

Comment:
Is it intentional to have space characters before the `.*` here? It might limit the matching e.g. if the command is the last custom command in the `customFFmpegOptions` string